ci-jobs/scripts/start-emulator.sh

```diff
@@ -16,32 +21,39 @@ echo $ANDROID_HOME/emulator/emulator -list-avds
 echo "Starting emulator"
 
 # Start emulator in background
-nohup $ANDROID_HOME/emulator/emulator -avd $ANDROID_AVD -no-snapshot > /dev/null 2>&1 &
-
-$ANDROID_HOME/platform-tools/adb wait-for-device get-serialno
-secondsStarted=$(date +%s)
-TIMEOUT=360
-while [[ $(( $(date +%s) - secondsStarted )) -lt $TIMEOUT ]]; do
-  # Fail fast if Emulator process crashed
-  pgrep -nf avd || exit 1
-
-  processList=$(adb shell ps)
-  if [[ "$processList" =~ "com.android.systemui" ]]; then
-    echo "System UI process is running. Checking IME services availability"
-    $ANDROID_HOME/platform-tools/adb shell ime list && break
+nohup $ANDROID_HOME/emulator/emulator -avd $ANDROID_AVD -accel auto -no-boot-anim -no-snapshot > /dev/null 2>&1 &
+
+if [ $ANDROID_SDK_VERSION -eq 30 ]; then
+  # Emulator API Level 30 seems work only below on CI
+  ${ANDROID_HOME}/platform-tools/adb wait-for-device shell 'while [[ -z $(getprop sys.boot_completed | tr -d "'"\r"'") ]]; do sleep 1; done; input keyevent 82'
```

Comment:
Codacy found an issue: [Expressions don't expand in single quotes, use double quotes for that.](https://app.codacy.com/manual/dpgraham/appium-espresso-driver/pullRequest?prid=5731265)

Reply:
Follows old way for emulator 30 since below `else` clause seemed it did not wait for the emulator ready to interact with. We can remove here when the behaviour in future release, I think.
https://dev.azure.com/AppiumCI/Appium%20CI/_build/results?buildId=10256&view=logs&j=e9fa3c63-21ed-5730-b945-3cceafbe7f30&t=651403d0-3420-5784-3a62-c1b43ce9a6b6